### PR TITLE
Asynchronously load library details in asset store

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/asset-library/views/LibraryDetails/LibraryInfo.jsx [hunk 1]
```diff
@@ -146,17 +150,27 @@ function LibraryInfo(props) {
...
       </div>
       <div className="library-column-info">
-        <h1>{library.name}</h1>
+        <h1>{libraryName}</h1>
         {library.description && <h2>{t('label.description')}</h2>}
         {library.description && <p>{library.description}</p>}
         <div className="library-credits">
-          <CreditSection title={t('author')} users={[library.author]} />
-          <CreditSection title={t('credits')} users={library.credits} />
+          {library.author &&
```
Comment (mileswong) [inline src/asset-library/views/LibraryDetails/LibraryInfo.jsx]: add `library && `?

File: src/asset-library/views/LibraryDetails/LibraryInfo.jsx [hunk 2]
```diff
@@ -146,17 +150,27 @@ function LibraryInfo(props) {
...
+            <CreditSection
+              title={t('author')}
+              users={[library.author]}
+            />
+          }
+          {library.credits && library.credits.length > 0 &&
```
Comment (mileswong) [inline src/asset-library/views/LibraryDetails/LibraryInfo.jsx]: add `library && `?

Comment (nwingt) [reply src/asset-library/views/LibraryDetails/LibraryInfo.jsx]: Updated

File: src/asset-library/views/LibraryDetails/LibraryInfo.jsx (hunk 1 above)
Comment (nwingt) [reply src/asset-library/views/LibraryDetails/LibraryInfo.jsx]: Updated

**Pull request merged**